fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ResourceDAOImpl.java

```diff
@@ -549,6 +564,19 @@ public Resource insert(Resource resource, List<ExtractedParameterValue> paramete
 
             resource.setId(stmt.getLong(7));
 
+            if (large) {
+                // Use the long id to update the record in the database.
+                String largeStmtString = String.format(LARGE_BLOB, getSchemaName(), resource.getResourceType());
+                PreparedStatement ps = connection.prepareStatement(largeStmtString);
+                ps.setLong(1, stmt.getLong(7));
```

Comment:
where does it actually set the bytes for the BLOB column?!

Reply:
Thanks, silly me - I was doing this late at night